documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,136 @@
+
+.Cannot schedule migrated VM with multiple disks to more than one storage classes of type hostpath
+
+When migrating a VM with multiple disks to more than one storage classes of type hostpath, it may result in a VM that cannot be scheduled. Workaround: It is recommended to use shared storage on the target OCP cluster.
+
+.Deleting migrated VM does not remove PVC and PV
+
+When removing a VM that was migrated, its PVCs and PVs are not deleted. +
+Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
+
+.PVC deletion hangs after archiving and deleting migration plan
+
+When archiving and deleting a migration plan that failed, its PVCs and PVs are not deleted. Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-493[(MTV-493)]
+
+.VM with multiple disks may boot from non-bootable disk after migration
+
+VM with multiple disks that was migrated may not be able to boot on the target OCP cluster. +
+Workaround: set the boot order appropriately to boot from the bootable disk. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
+
+.Non-supported guest operating systems in warm migrations
+
+Warm migrations and migrations to remote OCP clusters from vSphere do not support all types of guest operating systems that are supported in cold migrations to the local OCP cluster. It is a consequence of using RHEL 8 in the former case and RHEL 9 in the latter case. +
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+
+[id="resolved-issues-24_{context}"]
+== Resolved issues
+
+This release has the following resolved issue:
+
+.Map IDE to SATA +
+CNV does not support IDE disk interface, so the disks are mapped to SATA. link:https://issues.redhat.com/browse/MTV-345[(MTV-345)]
+
+.Improve invalid/conflicting VM name handling
+
+Users can automate VM renaming during mogration, to fit RFC 1123. link:https://issues.redhat.com/browse/MTV-212[(MTV-212)]
+
+.Prevent locking user accounts due to incorrect credentials
+
+When a user specifies an incorrect password for RHV providers, he no longer gets locked in RHV. An error returns when the RHV manager is accessible when adding the provider. If the RHV manager is not accessible, the provider is added, but there would be not further attempt after failing, due to incorrect credentials. link:https://issues.redhat.com/browse/MTV-324[(MTV-324)]
+
+.Convert i440fx to q35
+
+BIOS of i440fx chipset is reverted to BIOS of q35. link:https://issues.redhat.com/browse/MTV-430[(MTV-430)]
```

Comment:
`BIOS of i440fx chipset is reverted to BIOS of q35` doesn't seem accurate, how about: `Migrations of virtual machines with i440fx chipset is now supported. The chipset is converted to q35 during the migration.`

Reply:
Changed to your suggestion